### PR TITLE
News page

### LINKED ISSUE #37: 'News' page
Opens when clicking to 'Новости' in the header
In Figma - News
Screenshot 2020-06-07 at 13 52 10

When clicking to 'Показать по тегам' opens a dialog for search above 'sadaka' dialog
In Figma - Search News with tags
Screenshot 2020-06-08 at 20 07 58

### PR DESCRIPTION
**Pull request by Ural-2001**
I made the template of the news page. But as I understood I should also realize the tags feature, so the news cards have to appear according to selected tags, right? I've tried to make it, but I had struggles. So I need some help in realizing it. You can check the layout of the page for now.

Closes #37

### CODE REVIEW

File: components/News/Sadaka.vue [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+    <v-card
+      flat
+      class="sadaka-card"
+      style="border-radius: 10px;"
+    >
+      <v-card-title>Совершите садака</v-card-title>
+      <v-card-text>Любое ваше благодеяние, совершенное искренне, мы распределим между нуждающимися</v-card-text>
+      <v-card-text class="mt-3">
+        <v-text-field
+          v-model.number="input"
```
Comment (aabounegm) [inline components/News/Sadaka.vue:12]: `input` is not defined anywhere on the component

File: components/News/TagsSlider.vue [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+            :label="'#'+tag"
+            @click="change(tag)"
+          />
+          <div class="">
+            <v-btn
+              text
+              color="red"
+              @click="remove_all()"
+            >
+              <v-icon small class="mr-1">mdi-delete</v-icon>
```
Comment (aabounegm) [inline components/News/TagsSlider.vue:73]: Figma uses `mdi-close`

File: components/News/TagsSlider.vue [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+          <v-checkbox
+            v-for="tag in tags"
+            :key="tag"
+            :label="'#'+tag"
+            @click="change(tag)"
+          />
+          <div class="">
+            <v-btn
+              text
+              color="red"
```
Comment (aabounegm) [inline components/News/TagsSlider.vue:70]: Figma uses `color="primary"` here

File: components/News/TagsSlider.vue [hunk 4]
```diff
@@ -0,0 +1,120 @@
...
+              @click="remove_all()"
```
Comment (aabounegm) [inline components/News/TagsSlider.vue:71]: The parenthesis can be omitted in the template when there are no parameters passed and it is not part of an expression
```suggestion
              @click="remove_all"
```

File: components/News/TagsSlider.vue [hunk 5]
```diff
@@ -0,0 +1,120 @@
...
+    align-center
+    justify-space-between
+    class="tags-container"
+  >
+    <v-sheet
+      class="mx-5"
+      max-width="700"
+      style="background-color: #f2f2f2;"
+    >
+      <v-slide-group>
```
Comment (aabounegm) [inline components/News/TagsSlider.vue:12]: Check out the `prev` and `next` slots of [`v-slide-group`](https://vuetifyjs.com/en/components/slide-groups/#api) for customizing the icon

File: components/News/TagsSlider.vue [hunk 6]
```diff
@@ -0,0 +1,120 @@
...
+            color="black"
+            class="sadaka-input"
+          />
+          <v-checkbox
+            v-for="tag in tags"
+            :key="tag"
+            :label="'#'+tag"
+            @click="change(tag)"
+          />
+          <div class="">
```
Comment (aabounegm) [inline components/News/TagsSlider.vue:67]: What is this div? Why does it have an empty class?

File: components/News/TagsSlider.vue [hunk 7]
```diff
@@ -0,0 +1,120 @@
...
+              Очистить
+            </v-btn>
+          </div>
+        </v-list>
+      </v-menu>
+    </div>
+  </v-layout>
+</template>
+
+<script>
```
Comment (aabounegm) [inline components/News/TagsSlider.vue:83]: Typescript

**Comment by Yasinaaa** [comment]
- [ ] Fix date format, it must be: 21 февраля 2020 13:00
Now is: Wed Jul 01 2020 08:56:01 GMT+0300 (Moscow Standard Time)

**Comment by Ural-2001** [comment]
I'm stuck with making the checkboxes and prev/next buttons for the tag slider look like in figma. Should I do them entirely custom or there is a way to make it with vuetify, unfortunately I didn't find it. Also, I have an issue with the date property of the news. I'm trying to use Date.prototype.toLocaleDateString in the template like this: {{ Date.UTC(2012, 11, 20, 3, 0, 0).toLocaleDateString('en-US', { weekday: 'long', year: 'numeric', month: 'long', day: 'numeric' }) }} but it gives an error saying it's not a function

**Comment by aabounegm** [comment]
The next/prev buttons you set are fine (approved by Yasina).
`Date.UTC` returns a number, so it doesn't have `toLocaleDateString`. I see you already fixed it, and it is almost correct. I'll just do a couple of minor adjustments and it should be good to go. Great work

**Pull request merged**